### ISSUE TITLE
Attribute Error with my First Ubermag notebook

### ISSUE THREAD

**Issue by will-colea** [ubermag/ubermag#104]
Hello, I reinstalled everything and ran `python3 -c "import oommfc; oommfc.test()"` and everything passed.

I am working on the first notebook and everything ran great until section 1.1.3. Driving the system. 
I tried running the cell below...
`md = oc.MinDriver()`
`md.drive(system)`

The error I received was...
AttributeError: 'Mesh' object has no attribute 'pbc' 

I did initially have problems with the discretisedfield during the installation, so I am wondering if this could be a related issue.

**Comment by marijanbeg** [comment]
Same issue we had. Make sure you are in ubermag-test environment and run:

``conda install --channel conda-forge discretisedfield==0.8.10``

**Comment by will-colea** [comment]
I made sure to do that. 

It may be an issue with me not running my notebook in the correct environment. I will work on this and check back.

**Comment by will-colea** [comment]
Hello, so I made the notebook in the ubermag-test environment that we created and it still failed with the same error: 
AttributeError: 'Mesh' object has no attribute 'pbc'

See aattached image.
![Screen Shot 2020-06-11 at 12 14 51 PM](https://user-images.githubusercontent.com/59571636/84424398-48757200-abdd-11ea-9fe2-c166f7dd9f6a.png)

**Comment by will-colea** [comment]
Just now I went into the ubermag-test environment and ran 
`conda install --channel conda-forge discretisedfield==0.8.10`

It failed... which I am confused about now. Here is the terminal view.
![issues](https://user-images.githubusercontent.com/59571636/84432313-e3744900-abe9-11ea-84d5-1a6ffc5e472b.png)

**Comment by marijanbeg** [comment]
Hi @will-colea, you are not in `ubermag-test` environment. `(base)` at the beginning of the line indicates you are in base environment.

The environment is activated using `conda activate ubermag-test`

**Comment by will-colea** [comment]
OK, I moved into the ubermag-test environment and then ran 
`conda install --channel conda-forge discretisedfield==0.8.10`

It was successful!

I tried again with my juptyer notebook at the failure point, but now it can't find oommf to run the simulation. See attached error.
![Screen Shot 2020-06-11 at 1 57 02 PM](https://user-images.githubusercontent.com/59571636/84433631-d22c3c00-abeb-11ea-9572-09154a38a35a.png)
![Screen Shot 2020-06-11 at 1 57 24 PM](https://user-images.githubusercontent.com/59571636/84433635-d3f5ff80-abeb-11ea-8922-3921574c40fc.png)
